Tensile/ClientExecutable.py

```diff
@@ -79,8 +79,8 @@ def clientExecutableEnvironment(builddir=None):
         builddir = os.path.join(globalParameters["OutputPath"], globalParameters["ClientBuildPath"])
     builddir = Common.ensurePath(builddir)
 
-    CxxCompiler = "clang++.exe" if ((os.name == "nt") and supportedLinuxCompiler(globalParameters['CxxCompiler'])) else globalParameters['CxxCompiler']
-    CCompiler   = "clang.exe"   if ((os.name == "nt") and supportedLinuxCompiler(globalParameters['CxxCompiler'])) else globalParameters['CCompiler']
+    CxxCompiler = "clang++.exe" if ((os.name == "nt") and supportedCompiler(globalParameters['CxxCompiler'])) else globalParameters['CxxCompiler']
+    CCompiler   = "clang.exe"   if ((os.name == "nt") and supportedCompiler(globalParameters['CxxCompiler'])) else globalParameters['CCompiler']
```

Comment:
Think the second "and supportedCompiler(globalParameters['CxxCompiler']) "should be C compiler 'CCompiler'.   Using the global var as a test for support but then ignoring it and using the hard coded one seems strange.